sdlib/d/gc/emap.d

```diff
@@ -41,6 +41,10 @@ public:
 	void clear(Extent* extent) shared {
 		tree.clearRange(extent.address, extent.size);
```

Comment:
Just call clear if you are adding that.

Reply:
I know, but now that you have a method to clear from a pointer and a size, forward to that one instead of reimplementing the same functionality inline.